tree/tree/src/TChain.cxx

```diff
@@ -1273,6 +1273,22 @@ Int_t TChain::LoadBaskets(Long64_t /*maxmemory*/)
    return 0;
 }
 
+namespace ROOT::Internal::TChain {
```

Comment:
I don't think we need to declare this namespace here, a simple unnamed namespace suffices

Reply:
You have a point. The unnamed namespace should also hide the symbol because of the internal linkage. Given that this symbol is effectively useless outside of this translation unit, this seems the right thing to do.